modules/ROOT/pages/about-logging-of-payload-formats.adoc

```diff
@@ -1,27 +1,30 @@
+= Payload Formats and Logging 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 :keywords: analytics, monitoring, splunk, elk, elastisearch, logstash, kibana, logs, mule events, logging, apy analytics, metrics, traceability, arm, anypoint runtime manager
 
-image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-In order to send message payload data xref:sending-data-from-arm-to-external-analytics-software.adoc[to external analytics software], it must first be serialized and converted to text.
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-Some payload formats can't be converted to plain text. Because of this, the Runtime Manager Agent differentiates formats accordingly and only converts those that are considered to work.
 
-When the payload format isn't of a convertible type, the agent only passes on the full qualified name of the payload class type, not its content.
+To send message payload data to external analytics software, Runtime Manager agent must first serialize and convert the data to text.
 
-[WARNING]
-The information on this document only applies as of version xref:release-notes::runtime-manager-agent/runtime-manager-agent-1.6.3-release-notes.adoc[1.6.3] of the Runtime Manager agent. Prior versions attempt to convert all formats, causing failures on some.
+Because some payload formats can't be converted to plain text, Runtime Manager agent only converts those formats that are considered to work. 
+When the payload format can't be converted, the agent passes only the fully qualified name of the payload class type, not the payload message content.
+
+[NOTE]
+Runtime Manager agent version 1.6.2 and earlier attempts to convert all payload formats, which results in failures for some formats.
 
 == Default Conversion Behavior
 
-The Runtime Manager agent only passes on payload messages where the *Converted to String* column reads `TRUE` in the table below:
+The following table lists the payload formats and whether Runtime Manager agent converts them to String and passes the payload message.
+If the *Converted to String* column shows `FALSE`, Runtime Manager agent passes only the payload class type.
```

Comment:
In this case, the `FALSE` refers to the value in the table (not GUI). Technically, this one should probably be regular font.

As for bold for GUI items, I have been trained to tag actual GUI text - for translation purposes. I know we don't have semantic markup, but this is a hard habit to break.

Because the UX style is Sentence case (although not really consistently followed), I believe the bold helps with scanning and comprehension because it distinguishes the UI element in the middle of a sentence. 

Finally, I think we need to simplify our style guidelines. The fewer things writers need to keep in our heads, the easier it is to follow the style consistently (and ultimately less work for you and peer reviewers). So, for example, a guideline that's "bold all UI elements" is much easier to remember than "bold UI elements only when instructing the user to click or select". Another example of style simplification would be "Always include terminating punctuation in list items". That eliminates the whole "is this a complete sentence or not?" and "What do I do if I have a complete sentence after the sentence fragment?" lookups (or not) in the SG.

My (lofty) goal is to have the style in my head, so I don't have to look stuff up.

Reply:
I fixed the FALSE. I supposed in the olden days, this would be "FALSE".

Reply:
Yeah, I hear you about making the style rules simpler than English grammar and punctuation, with all its deviations and exceptions. Thanks for reminding me that my second most important goal is making writers' lives easier....
I'm really sorry that UX decided on sentence-style labeling in the UI. It might look better there, but it introduces this very problem for docs. I conversed a bit with Tony about it, who agrees but indicated that consensus made this a done deal.